itou/job_applications/migrations/0014_copy_planned_training_days_data.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.1.9 on 2023-06-08 09:39
+
+from django.db import migrations
+
+from itou.job_applications.models import JobApplication
+
+
+def copy_planned_days_to_planned_hours(apps, schema_editor):
+    for ja in JobApplication.objects.filter(planned_training_days__isnull=False):
+        ja.planned_training_hours = ja.planned_training_days
+        ja.save(update_fields=["planned_training_hours"])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("job_applications", "0013_jobapplication_planned_training_hours"),
+    ]
+
+    operations = [migrations.RunPython(copy_planned_days_to_planned_hours)]
```

Comment:
Pour être gentil avec la personne qui fera le squash de migration :).
```suggestion
    operations = [migrations.RunPython(copy_planned_days_to_planned_hours, reverse_code=migrations.RunPython.noop, elidable=True)]
```

Reply:
Très intéressant. TIL `elidable` sur la migration.

Mais on peut renommer le champs en une migration: renommage du champ et MaJ de la description en deux opérations séquentielles.

Pour la cohérence des données, les gens ont bien saisi des heures de formation et pas des jours, c'est juste l'intitulé qui est faux. Mais sinon, oui j'aurais du envisager ce genre de conversions ...